substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/NativeImageOptions.java

```diff
@@ -60,7 +60,7 @@
     public static final HostedOptionKey<Boolean> NativeArchitecture = new HostedOptionKey<>(false);
 
     @Option(help = "Define PageSize of a machine that runs the image. The default = 0 (== same as host machine page size)")//
-    protected static final HostedOptionKey<Integer> PageSize = new HostedOptionKey<>(0);
+    public static final HostedOptionKey<Integer> PageSize = new HostedOptionKey<>(0);
```

Comment:
I'm not clear where the need for making this public this arose? Can you point at where this is used?

Reply:
This doesn't belong in this PR, sorry.  At the time it seemed inconsistent, but this isn't the place for it.  Removed.